sgt-back-end/index.jsx

```diff
@@ -0,0 +1,140 @@
+const express = require('express');
+const pg = require('pg');
+const app = express();
+
+const db = new pg.Pool({
+  connectionString: 'postgres://dev:lfz@localhost/studentGradeTable'
+});
+
+app.use(express.json());
+
+app.get('/api/grades', (req, res, next) => {
+  const sql = `
+    select *
+      from "grades"
+  `;
+  db.query(sql)
+    .then(result => {
+      const grades = result.rows;
+      if (!grades) {
+        res.status(500).json({
+          error: 'We are currently unable to handle your reguest'
+        });
```

Comment:
The 500 response should be done in the `.catch` of your db.query in case the promise is rejected.